docs/style-guide-for-curriculum-challenges.md

```diff
@@ -124,6 +124,15 @@ Here are specific formatting guidelines for challenge text and examples:
 - Language keywords go in `<code>` tags. For example, HTML tag names or CSS property names
 - The first instance of a keyword when it's being defined, or general keywords (i.e. "object" or "immutable") go in `<dfn>` tags
 - Single line code examples go in `<code>` tags
```

Comment:
I think this line needs to be changed, because it is not that you have to use `<code>` tags for one line code segments.  It is for surrounding very short code (i.e. variable names, function/method names).  I think showing an example is best here.